### ISSUE TITLE
[BUG] Change python3 to python

### ISSUE THREAD

**Issue by mxschmitt** [EngFlow/reclient-configs#7]
If you install Python on a Windows machine, it usually is only available as `python` and not as `python3`. This will then e.g. break the following: https://github.com/EngFlow/reclient-configs/blob/fe37dd84dc35cc9524f27e5b281b5d896c22fd8a/configure_reclient.py#L144

We can probably expect nowadays machines have Python3 as `python`. Would you be up for a contribution to change `python3` -> `python`?

Happy for more thoughts on this, thanks!

**Comment by ola-rozenfeld** [comment]
Hmm, we have Chromium builds on Windows, and what worked was running both scripts with `cmd`, e.g.:

```
cmd.exe /c "python3 reclient-configs\configure_reclient.py --src_dir=%SRC_PATH%"
```

So Windows can run `python3`, it just needs some convincing :-)

I'm a wary that the change might backfire with existing users, e.g. [Brave](https://github.com/brave/brave-core/blob/7fd1c6f3c09e41b173a7654cb0cf2d4acc9d91d1/DEPS#L71). To make it backward compatible we'd probably need to clone both scripts and have a `python` version alongside a `python3` version, which is just not quite worth it, imo.

**Comment by mxschmitt** [comment]
hm weird! So when I installed Python [officially](https://www.python.org/downloads/) there was only `python` and no `python3` in my PATH. Sounds like https://github.com/python/cpython/issues/99185.

TL;DR: `python3` will target the WindowsStore Python while `python` will target the actual `python.exe`.

I have a workaround for now, so happy to close, just confusing and I thought python is what tools use nowadays.

What convention does Chromium use upstream? Looks like `python3`.

**Comment by ola-rozenfeld** [comment]
> What convention does Chromium use upstream? Looks like python3.

AFAIK, Chromium migrated to `python3` everywhere and now relies on the Python version to be at least 3.8.

However: I just realized that I think I can fix your problem with just this: https://github.com/EngFlow/reclient-configs/pull/8
Can you try this out, please? Thank you!

**Comment by mxschmitt** [comment]
I can confirm that this works for us. Appreciate!
